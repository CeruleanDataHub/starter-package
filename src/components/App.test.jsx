import React from "react";
import { mount } from "enzyme";

import App from "./App";

describe("App", () => {
    let component;

    beforeEach(() => {
        component = mount(<App />);
    });

    it("renders", () => {
        expect(component).toMatchSnapshot();
    });
});
