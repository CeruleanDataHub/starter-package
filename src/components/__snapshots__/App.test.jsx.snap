// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders 1`] = `
<App>
  <div>
    Hello World
  </div>
</App>
`;
